android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java

```diff
@@ -91,11 +81,44 @@
     @BindView(R.id.speaker_details_designation) TextView speakerDesignation;
     @BindView(R.id.progress_bar)
     protected ProgressBar progressBar;
+    private CustomTabIntent customTabIntent;
+
+    @OnClick({R.id.imageView_linkedin, R.id.imageView_fb, R.id.imageView_github, R.id.imageView_twitter, R.id.imageView_web})
+    public void openUrl(View v) {
+        int id = v.getId();
+        String url;
+        switch (id) {
+            case R.id.imageView_linkedin:
+                url = speaker.getLinkedin();
+                break;
+            case R.id.imageView_fb:
+                url = speaker.getFacebook();
+                break;
+            case R.id.imageView_github:
+                url = speaker.getGithub();
+                break;
+            case R.id.imageView_twitter:
+                url = speaker.getTwitter();
+                break;
+            case R.id.imageView_web:
+                url = speaker.getWebsite();
+                break;
+            default:
+                return;
+        }
+
+        if (!TextUtils.isEmpty(url)) {
+            if (customTabIntent == null)
+                customTabIntent = new CustomTabIntent(this);
```

Comment:
Hi @hardik124 Nice work :clap: 
Here No need to create new class `CustomTabIntent`. You can directly use Util method we have already created https://github.com/fossasia/open-event-android/blob/development/android/app/src/main/java/org/fossasia/openevent/utils/Utils.java#L110.

Reply:
Hi @Shailesh351 . Thanks. I didnt know about the Utils class. I will fix it ASAP.
In the original custom tab method(Utils). There is no check if custom tabs isnt available, or if URL is valid or not. Should I use my code here to replace that ?

Reply:
Yes. Add check for URL is valid or not. For availability of CustomTabs i think it will be handled automatically if not available it will start intent.